src/System.ServiceModel.Duplex/src/System.ServiceModel.Duplex.csproj

```diff
@@ -19,10 +19,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Release|AnyCPU'" />
 
   <ItemGroup>
-    <TargetingPackReference Include="mscorlib" Condition="'$(TargetGroup)' != 'netcore50aot'" />
-    <TargetingPackReference Include="System.ServiceModel" Condition="'$(TargetGroup)' == 'net46'" />
-    <ProjectReference Include="$(SourceDir)System.Private.ServiceModel\src\System.Private.ServiceModel.csproj" Condition="'$(TargetGroup)' != 'net46'" >
-      <AdditionalProperties>OSGroup=Windows_NT</AdditionalProperties>
+    <ProjectReference Include="$(SourceDir)System.Private.ServiceModel\src\System.Private.ServiceModel.csproj" >
```

Comment:
I see, you are [defaulting the OSGroup in the S.P.SM project](https://github.com/dotnet/wcf/blob/master/src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj#L4).  In that case it won't build twice but it will be a race condition between the project reference (missing OSGroup global property) and the builds-file reference (with the OSGroup global property).
I've been working on cleaning these up as well as adding a logger to detect them.